spec/spec.md

```diff
@@ -865,7 +871,10 @@ Descriptor Objects_ are composed as follows:
         ****MUST**** be an array consisting of one or more valid URI strings for
         the acceptable credential schemas. A common use of multiple entries in
         the `uri` array is when multiple versions of a credential schema exist
-        and you wish to express support for submission of more than one version.
+        and you wish to express support for submission of more than one version. 
+        This field allowing multiple URIs ****IS NOT**** intended to be used as 
```

Comment:
should ****IS NOT**** be a normalized term here?

Reply:
I just wanted to really emphasize it, but we don't have to if you don't think it's right to do that.